frontend/src/components/Loadable/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { lazy, ComponentType } from "react";
+
+function Loadable<T>(importPath: {
+	(): LoadableProps;
+}): (props: T) => JSX.Element {
+	const LazyComponent = lazy(() => importPath());
+
+	return () => <LazyComponent />;
```

Comment:
We can just do `return LazyComponent`

Reply:
@jyash97 let say if we want to return `LazyComponent` rather than `()=> < LazyComponent />`

then I have to change in the [AppRoutes](https://github.com/palash-signoz/signoz-1/blob/6e586d55ed09b665c57498cc580a303b611b2e30/frontend/src/AppRoutes/routes.ts#L64) with component `(()=>JSX.Element)` rather than `JSX.Element`

Let me know if this is change we are looking for

Reply:
but i would prefer`()=>JSX.Element`

Reply:
updated the code so that the function will return `React.LazyExoticComponent`

in which will not return anonymous function

Thanks @jyash97